dgraph/cmd/alpha/login_ee.go

```diff
@@ -1,4 +1,4 @@
-// +build !oss
+//go:build !oss
```

Comment:
This is some auto fmt from latest goimports. Not sure what it does. If you have a go server with goimports installed. Everytime you save the file it will update that comment.

There are few expressions with this
```
!dev
!oss (which means "open source software" as far as I know. Also it could be "cross platform OS")
!debug
!windows
```

What this fmt does is just add `go:` in whatever it finds.

I've just chose not fight against it. Cuz it is something that will update no matter what I do against it.